lua/wikis/commons/Infobox/Person.lua

```diff
@@ -349,6 +426,23 @@ function Person:adjustLPDB(lpdbData, args, personType)
 	return lpdbData
 end
 
+---@return string
+function Person:_isPlayerOrStaff()
+	local inGameRoles = Roles.InGameRoles
+
+	local roles = Logic.nilIfEmpty(self.roles) or {self.role, self.role2, self.role3}
+
+	if Logic.isEmpty(roles) then return 'player' end
+
+	local isPlayer = Array.any(roles, function(roleData)
+		return Table.any(ROLES, function(roleKey, data)
+			return data == roleData and inGameRoles and inGameRoles[roleKey]
+		end)
+	end)
+
+	return isPlayer and 'player' or 'staff'
+end
```

Comment:
Shouldn't this be merged with getPersonType?
Now it's possible to have one function treating someone as staff, and another as player.